runtime/src/main/java/org/corfudb/runtime/LogReplicationUtils.java

```diff
@@ -63,6 +65,15 @@ public final class LogReplicationUtils {
 
     // ---- End RoutingQueue Model constants -------/
```

Comment:
nit: can you move this comment after the constant definitions?

Reply:
done - will add it in a subsequent PR